components/camel-nats/src/main/java/org/apache/camel/component/nats/NatsConfiguration.java

```diff
@@ -90,6 +90,12 @@ public class NatsConfiguration {
     private boolean traceConnection;
     @UriParam(label = "advanced")
     private HeaderFilterStrategy headerFilterStrategy = new DefaultHeaderFilterStrategy();
+    @UriParam(label = "advanced", defaultValue = "true")
+    private boolean jetstreamEnabled = true;
```

Comment:
I don't think the default value should be true. It's not sure people will use jetstream by default.

Reply:
Totally agree!
I shall change it.